content/content-strategy/understand-content-ecosystem/show-content-ecosystem/mind-maps.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: blocks/image_content_landscape
+image: content-strategy/ecosystem/bad-ecosystem-map.jpg
+link: content-strategy/ecosystem/bad-ecosystem-map.jpg
+title: Mind maps
+reverse: false
+stackPosition: top
+imageAlt: A confusing and complex content ecosystem mind map showing many duplicate channels.
+caption: This complex ecosystem map shows the main website at the centre, surrounded by four content channel types, websites, email newsletters, social media, and apps. The channels are interconnected in a complicated and confusing way, and there are many subsets with duplicate channels.
+level: 3
+display: 3
```

Comment:
@annekemyers was this a deliberate choice to markup the heading as an h2 even though it appears as an h3? I've noticed some inconsistent heading levels and display values and just wanna understand the reason for the unusual `display` values. 
This one would surely be `display:2`?